modules/aws_roles/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable cluster_id {
+    description = "cluster ID"
+    type = string
+}
+
+variable rh_oidc_provider_url {
+    description = "oidc provider url"
+    type = string
+    default = "rh-oidc.s3.us-east-1.amazonaws.com"
+}
+
+variable rh_oidc_provider_thumbprint {
+    description = "Thumbprint for https://rh-oidc.s3.us-east-1.amazonaws.com"
```

Comment:
Does the description depend on the `rh_oidc_provider_url` value?

Reply:
I changed it. 
thanks